### ISSUE TITLE
Unit tests / functional tests

### ISSUE THREAD

**Issue by xenide** [dfinity/cancan#34]
Are there any tests written for this project? Can't seem to find it within the repo

**Comment by matthewhammer** [comment]
Yes, there are backend-only tests that exercise the logic there:
 - They run in CI, from here https://github.com/dfinity/cancan/blob/031f31c0f45af72e42416043e1a2415642844d4e/.github/workflows/backend.yml#L36
  - They use [`ic-repl`](https://github.com/chenyan2002/ic-repl), which I recommend trying if you have any tests to author OR if you want to interact with your canister using a REPL interface, for interactive diagnostics and tests.
  - [The ic-repl test scripts are here (as mentioned in CI script from first link)](https://github.com/dfinity/cancan/tree/main/service/tests).
